setup.rst

```diff
@@ -436,8 +448,8 @@ To overcome this problem, auto-generated files are also checked into the
 Git repository. So if you don't touch the auto-generation scripts, there's
 no real need to auto-generate anything.
 
-Editors and Tools
-=================
+Configure your editor
+=====================
```

Comment:
I think the previous title was fine even if it didn't include a verb (I'm assuming you added it for consistency with the other titles).  This section doesn't say much about configuring your editor.